### DISCUSSION TITLE
External updates to apps.json and triggering reload

### DISCUSSION THREAD

**Discussion by doxxx** [LizardByte/community#555] [Sunshine] [answered]
### Select Topic Area

Question

### Body

I'm writing a tool that manages installation of games from zips and registering them with Sunshine. Currently it can update `apps.json` with the newly installed game and it shows up in the Applications list on the Sunshine web app. However, the new game does not show up in the Moonlight client until I edit and save any defined application through the web app.

Is there an API to make the Sunshine server, not just the web app, reload the list of applications from `apps.json`?

**Comment by ReenigneArcher** [comment, marked as answer]
Here's the beta API documentation: https://docs.lizardbyte.dev/projects/sunshine/en/master/md_docs_2api.html
